app/views/authentication_mailer/sign_in_without_account_email.text.erb

```diff
@@ -0,0 +1,8 @@
+# <%= t('authentication.sign_in_without_account.email.subject') %>
+
+You attempted to sign in to the <%= t('service_name.apply') %>, but we could not find an account with your email address.
+
+Please click the link below to sign up.
```

Comment:
```suggestion
Click the link below to sign up to the <%= t('service_name.apply') %> service.
```

For similar content to `app/views/authentication_mailer/sign_up_email.text.erb`? Maybe...

Reply:
Sounds good! 😸 
Updated in: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/729/commits/41570c342d073489c899367013eda80e62cce531